mappings/net/minecraft/block/entity/SculkShriekerWarningManager.mapping

```diff
@@ -29,7 +29,7 @@ CLASS net/minecraft/class_7262 net/minecraft/block/entity/SculkShriekerWarningMa
 	METHOD method_42257 (Ljava/util/List;Lnet/minecraft/class_7262;)V
 		ARG 1 manager
 	METHOD method_42258 reset ()V
-	METHOD method_42259 isNoWardenInRange (Lnet/minecraft/class_3218;Lnet/minecraft/class_2338;)Z
+	METHOD method_42259 canIncreaseWarningCount (Lnet/minecraft/class_3218;Lnet/minecraft/class_2338;)Z
```

Comment:
I think you renamed this to "warning level"?

Reply:
ah yeah, good catch, renamed it that because it's called that in the codec
```suggestion
	METHOD method_42259 canIncreaseWarningLevel (Lnet/minecraft/class_3218;Lnet/minecraft/class_2338;)Z
```